ouroboros-consensus-cardano/tools/db-analyser/Analysis.hs

```diff
@@ -185,6 +191,7 @@ instance TPraosCrypto c => HasAnalysis (CardanoBlock c) where
   blockTxSizes blk = case blk of
     Cardano.BlockByron b    -> blockTxSizes b
     Cardano.BlockShelley sh -> blockTxSizes sh
+  knownEBBs = undefined
```

Comment:
@edsko I guess I should also fix this?

Reply:
Oh, sorry, I forgot about that. I can fix it if you want, or you can give it a go? It would only require a type translation/injection.

Reply:
Whichever you prefer, let me know.